src/components/widgets/system/SystemOverviewCard.vue

```diff
@@ -7,42 +7,48 @@
       <v-col>
         <v-simple-table dense>
           <tbody>
-            <tr>
+            <tr v-if="printerInfo.hostname">
               <th>{{ $t('app.system_info.label.hostname') }}</th>
               <td>{{ printerInfo.hostname }}</td>
             </tr>
-            <tr>
+            <tr v-if="cpuInfo.model">
               <th>{{ $t('app.system_info.label.model') }}</th>
               <td>{{ cpuInfo.model }}</td>
             </tr>
-            <tr>
+            <tr v-if="cpuInfo.cpu_desc">
               <th>{{ $t('app.system_info.label.cpu_desc') }}</th>
               <td>{{ cpuInfo.cpu_desc }}</td>
             </tr>
             <tr v-if="cpuInfo.total_memory">
               <th>{{ $t('app.system_info.label.total_memory') }}</th>
               <td>{{ $filters.getReadableFileSizeString(cpuInfo.total_memory * 1000) }}</td>
             </tr>
-            <tr>
+            <tr v-if="cpuInfo.hardware_desc">
               <th>{{ $t('app.system_info.label.hardware_desc') }}</th>
               <td>{{ cpuInfo.hardware_desc }}</td>
             </tr>
-            <tr>
+            <tr v-if="cpuInfo.bits && cpuInfo.processor && cpuInfo.cpu_count">
               <th>{{ $t('app.system_info.label.processor_desc') }}</th>
               <td>{{ cpuInfo.bits }} {{ cpuInfo.processor }} with {{ cpuInfo.cpu_count }} cores</td>
             </tr>
-            <tr>
-              <th>{{ $t('app.system_info.label.distribution_name') }}</th>
+            <tr v-if="distribution.name">
+              <th>{{ $t('app.system_info.label.operating_system') }}</th>
```

Comment:
Everything on the PR is perfect, only this label change here is giving me some doubts... I'm not sure what is the correct denomination here, if "Operating System" or "Distribution", but that comes from my lack of knowledge of Linux!

Reply:
I think operating system is correct because freedesktop.org  also calls it like this.

https://www.freedesktop.org/software/systemd/man/os-release.html

![image](https://user-images.githubusercontent.com/44060099/214277680-66e337ad-cea4-4f88-a729-bffe3731d29d.jpeg)

Reply:
Good enough for me!! 😁